sonar-db/src/main/java/org/sonar/db/issue/IssueChangeDao.java

```diff
@@ -93,17 +93,11 @@ public void insert(DbSession session, IssueChangeDto change) {
     mapper(session).insert(change);
   }
 
-  public boolean delete(String key) {
-    DbSession session = mybatis.openSession(false);
-    try {
-      IssueChangeMapper mapper = mapper(session);
-      int count = mapper.delete(key);
-      session.commit();
-      return count == 1;
-
-    } finally {
-      MyBatis.closeQuietly(session);
-    }
+  public boolean delete(DbSession session, String key) {
+    IssueChangeMapper mapper = mapper(session);
+    int count = mapper.delete(key);
+    session.commit();
+    return count == 1;
```

Comment:
It's used in bulk I think, but I agree I'd like to remove it.
Unfortunately not enough time to investigate for the moment.